app/src/main/java/org/groundplatform/android/usecases/survey/ListAvailableSurveysUseCase.kt

```diff
@@ -57,8 +57,11 @@ constructor(
   private suspend fun getRemoteSurveyList(): Flow<List<SurveyListItem>> {
     val user = userRepository.getAuthenticatedUser()
     val remoteSurveyListFlow = remoteDataStore.getSurveyList(user)
-    return remoteSurveyListFlow.combine(getLocalSurveyList()) { remoteSurveys, localSurveys ->
-      remoteSurveys.map { remoteSurvey -> addOfflineStatus(remoteSurvey, localSurveys) }
+    val localFlow = getLocalSurveyList()
+    return combine(remoteSurveyListFlow, localFlow) { remoteSurveys, localSurveys ->
+      val enrichedRemote = remoteSurveys.map { remote -> addOfflineStatus(remote, localSurveys) }
```

Comment:
Can we improve the name of the variable? Maybe something more explicit like `remoteSurveysWithOfflineStatus`

Reply:
fixed